articles/multifactor-authentication/index2.md

```diff
@@ -15,49 +15,45 @@ Multifactor Authentication (MFA) is a method of verifying a user's identity by r
 
 Auth0 supports the following methods of implementing MFA:
```

Comment:
I marked pr #1466 as do not merge so it doesnt get merged first and mess up this one :)